### ISSUE TITLE
Should the `examples` folder be removed?

### ISSUE THREAD

**Issue by janssenhenning** [JuDFTteam/aiida-fleur#157]

**Comment by janssenhenning** [comment]
Definitely when we have a nice structure for tutorials (judft_tutorials is a good start) this can be removed.

Ultmiately I think such examples should be nicely included in the docs

**Comment by Tseplyaev** [comment]
I agree, the submission script examples are basically duplicated in the documentation. I think we should keep them in the documentation only.
